catalog/transformer/catalog-transformer-streaming-impl/src/test/java/org/codice/ddf/transformer/xml/streaming/impl/Gml3ToWktImplTest.java

```diff
@@ -83,4 +92,92 @@ public void testConvertEnvelope() throws ValidationException {
     String wkt = gtw.convert(gml);
     assertThat(wkt, equalTo("POLYGON ((0 0, 0 1, 1 1, 1 0, 0 0))"));
   }
+
+  @Test
+  public void testGmlPointToWktValidSrsName() throws ValidationException {
+    assertThat(gtw.convert(createSrsAwareGmlPoint(0, 0, "EPSG:4326")), is(createWktPoint(0, 0)));
+    assertThat(gtw.convert(createSrsAwareGmlPoint(0, 1, "EPSG:4326")), is(createWktPoint(0, 1)));
+    assertThat(gtw.convert(createSrsAwareGmlPoint(1, 0, "EPSG:4326")), is(createWktPoint(1, 0)));
+    assertThat(gtw.convert(createSrsAwareGmlPoint(1, 1, "EPSG:4326")), is(createWktPoint(1, 1)));
+  }
+
+  @Test
+  public void testGmlPointToWktValidSrsNameUrn() throws ValidationException {
+    assertThat(
+        gtw.convert(createSrsAwareGmlPoint(0, 0, "urn:x-ogc:def:crs:EPSG:4326")),
```

Comment:
❓ does the "x-ogc" here indicate axis ordering for the 4326 CRS?  Should we be testing other 4326 URNs?

https://github.com/deegree/deegree3/wiki/Axis-order-handling, eg:
```
urn:ogc:def:crs:epsg::4326
urn:x-ogc:def:crs:epsg:6.11.2:4326
```